client/me/purchases/remove-plan-dialog/get-plan-features.tsx

```diff
@@ -0,0 +1,257 @@
+import {
+	isMonthly,
+	isWpComBusinessPlan,
+	isWpComEcommercePlan,
+	isWpComPersonalPlan,
+	isWpComPremiumPlan,
+} from '@automattic/calypso-products';
+import { useTranslate } from 'i18n-calypso';
+
+export default function getPlanFeatures(
+	productSlug: string | undefined,
+	translate: ReturnType< typeof useTranslate >,
+	withinFirstYear: boolean,
+	hasDomain: boolean,
+	domainSlug: string
+): string[] {
+	if ( ! productSlug ) {
+		return [];
+	}
+
+	const isMonthlyPlan = isMonthly( productSlug );
+
+	/**
+	 * Define feature labels.
+	 */
+	const freeDomainFirstYear = String( translate( 'Your free domain for the first year' ) );
+	const freeDomain = String(
+		translate( 'Your free domain www.%(slug)s', {
+			args: { slug: domainSlug },
```

Comment:
I don't really understand the above wording.... We don't take away anyone's domain when they cancel the plan.  We might subtract its **cost** from their refund (if the plan is within the refund window and the domain isn't) but that would be in the refund flow, which is different than the main flow you're in here.

Reply:
domains labels and logic were removed in the design vTj2aknM6tRUQndWtOcURT-fi-445%3A10488